src/apps/bridge/sensor_drivers/rbrCodaSensor.h

```diff
@@ -31,10 +31,10 @@ typedef struct RbrCodaSensor : public AbstractSensor {
   // 2 minutes is the minimum bridge on period and the soft by default is sampling at 2Hz. So 2*120 + 30 = 270.
   static constexpr uint32_t N_SAMPLES_PAD = 270;
   static constexpr uint8_t MIN_READINGS_FOR_AGGREGATION = 3;
-  static constexpr double TEMP_SAMPLE_MEMBER_MIN = -5;
-  static constexpr double TEMP_SAMPLE_MEMBER_MAX = 50;
+  static constexpr double TEMP_SAMPLE_MEMBER_MIN = -20;
+  static constexpr double TEMP_SAMPLE_MEMBER_MAX = 84.8572;
   static constexpr double PRESSURE_SAMPLE_MEMBER_MIN = 5;
-  static constexpr double PRESSURE_SAMPLE_MEMBER_MAX = 85;
+  static constexpr double PRESSURE_SAMPLE_MEMBER_MAX = 200;
```

Comment:
The are the min and max for _aggregation_. Our specification allows pressure sample mean values up to 20 bar (not 8.5). We allow temperature sample mean values from -20ºC to about 84.8ºC.